src/test/resources/features/login/login.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Login for trello
+
+  In order to login in trello
+
+  Scenario: I am trying login with the valid credentials
```

Comment:
Please be consistent with the title of your scenarios, you should keep a format, also add more verifications please